src/getAll.js

```diff
@@ -1,23 +1,41 @@
 import ColorSpace from "./ColorSpace.js";
 import getColor from "./getColor.js";
+import { toPrecision, type } from "./util.js";
 
 /** @typedef {import("./types.js").ColorTypes} ColorTypes */
 /** @typedef {import("./types.js").Coords} Coords */
 
 /**
  * Get the coordinates of a color in any color space
  * @param {ColorTypes} color
- * @param {string | ColorSpace} space The color space to convert to. Defaults to the color's current space
+ * @param {string | ColorSpace | object} [options=color.space] If a string or ColorSpace, the color space to convert to. Defaults to the color's current space.
+ *                                       If an object:
+ *                                       [options.space=color.space] — the color space to convert to (defaults to the color's current space);
+ *                                       [options.precision] — the number of significant digits to round the coordinates to.
```

Comment:
Unfortunately I can't select the entire JSDoc and add a suggestion because you can't suggest on deleted lines, but for now I’ll just add a suggestion for this:

```suggestion
 * @param {string | ColorSpace | GetAllOptions} [options=color.space] If a string or ColorSpace, the color space to convert to. Defaults to the color's current space.
```

Reply:
Yeah, I quite often need this feature to suggest changes to lines that are not in the PR.